mathics/core/atoms.py

```diff
@@ -381,22 +379,29 @@ def user_hash(self, update):
         update(b"System`Real>" + str(self.to_sympy().n(dps(_prec))).encode("utf8"))
 
 
+# Has to come before PrecisionReal
 class MachineReal(Real):
     """
     Machine precision real number.
 
     Stored internally as a python float.
     """
 
-    value: float
```

Comment:
Why remove this?

Reply:
In line 395 it looks like `value` is an _instance_ variables not a _class_ variable.

Reply:
Instance variables also can be annotated:
https://peps.python.org/pep-0526/#class-and-instance-variable-annotations
Class anotations are only considered class variables when they have a initial value.

Reply:
Thanks I was not aware of this.